ui/apps/platform/src/Containers/PolicyCategories/DeletePolicyCategoryModal.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+    Modal,
+    ModalVariant,
+    ModalBoxBody,
+    ModalBoxFooter,
+    Button,
+    List,
+    ListItem,
+    ListComponent,
+    OrderType,
+    Flex,
+    FlexItem,
+    Panel,
+    PanelMain,
+    PanelMainBody,
+} from '@patternfly/react-core';
+
+import { getPolicies } from 'services/PoliciesService';
+import { deletePolicyCategory } from 'services/PolicyCategoriesService';
+import { PolicyCategory, ListPolicy } from 'types/policy.proto';
+import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+
+type DeletePolicyCategoryModalType = {
+    isOpen: boolean;
+    onClose: () => void;
+    addToast: (toast) => void;
+    refreshPolicyCategories: () => void;
+    selectedCategory?: PolicyCategory;
+    setSelectedCategory: (category?: PolicyCategory) => void;
+};
+
+function DeletePolicyCategoryModal({
+    isOpen,
+    onClose,
+    addToast,
+    refreshPolicyCategories,
+    selectedCategory,
+    setSelectedCategory,
+}: DeletePolicyCategoryModalType) {
+    const [affectedPolicies, setAffectedPolicies] = useState<ListPolicy[]>([]);
+
+    function handleDelete() {
+        deletePolicyCategory(selectedCategory?.id || '')
+            .then(() => {
+                addToast('Successfully deleted category');
+                setSelectedCategory();
+                refreshPolicyCategories();
+            })
+            .catch((error) => {
+                addToast(error.message);
+            })
+            .finally(() => {
+                onClose();
+            });
+    }
+
+    useEffect(() => {
+        const query = getRequestQueryStringForSearchFilter({ Category: selectedCategory?.name });
+        getPolicies(query)
+            .then((policies) => {
+                setAffectedPolicies(policies);
+            })
+            .catch((error) => {
+                addToast(error.message);
+            });
+    }, []);
+
+    return (
+        <Modal
+            title="Permanently delete category?"
+            isOpen={isOpen}
+            variant={ModalVariant.small}
+            onClose={onClose}
+            data-testid="delete-category-modal"
+            aria-label="Permanently delete category?"
+            hasNoBodyWrapper
+        >
+            <ModalBoxBody>
+                {affectedPolicies.length > 0 && (
+                    <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                            This will permanently delete and remove the
+                            <b> {selectedCategory?.name} </b>
+                            policy category from the following policies:
+                        </FlexItem>
+                        <FlexItem>
+                            <Panel variant="bordered" isScrollable>
+                                <PanelMain>
+                                    <PanelMainBody>
+                                        <List component={ListComponent.ol} type={OrderType.number}>
```

Comment:
according to the pf docs, this should render a numbered list (https://www.patternfly.org/v4/components/list#ordered) but it is not showing the numbers for me. i'll continue to look into it, but didn't want it to hold up this PR.

Reply:
Tailwind has a reset style rule for lists. Here is an override that I tested for src/css/trumps.css file:

```css
/* Override none value for list-style in Tailwind stylesheet. */
.pf-c-modal-box ol,
.pf-c-page__main-section ol {
    list-style: decimal;
}
```